### PR TITLE
roothash/api: rename Genesis.Blocks -> RuntimeStates

### PR DESCRIPTION
**Pull request by matevz**
oasis-runtime PR for https://github.com/oasislabs/oasis-core/issues/2426
Merge after PR https://github.com/oasislabs/oasis-core/pull/2462 has been merged in oasis-runtime

### CODE REVIEW

**Comment by matevz** [comment]
This one is blocked by https://github.com/oasislabs/oasis-runtime/issues/962

**Comment by ptrus** [comment]
@matevz this should be now unblocked :) let me know if you need any help

File: genesis/bin/genesis_playback.rs [hunk 1]
```diff
@@ -166,13 +178,14 @@ fn main() {
                 .long("runtime-id")
                 .takes_value(true)
                 .required(true)
-                // TODO: Remove this default value to discourage use (see ekiden#1693).
-                .default_value("0000000000000000000000000000000000000000000000000000000000000000"),
+                // TODO: Remove this default value to discourage use (see oasis-core#1693).
+                .default_value("8000000000000000000000000000000000000000000000000000000000000000"),
```
Comment (kostko) [inline genesis/bin/genesis_playback.rs]: Can we just remove the default or will it break something (e.g., benchmarks)?

File: genesis/bin/genesis_playback.rs [hunk 2]
```diff
@@ -297,13 +310,20 @@ fn main() {
...
+        }],
+        round: 0,
+    };
+    let rtg_map: HashMap<roothash::Namespace, registry::RuntimeGenesis> =
+        [(runtime_id, rtg)].iter().cloned().collect();
 
     // Save to file.
     let mut file = File::create(matches.value_of("output_file").unwrap()).unwrap();
-    serde_json::to_writer(&mut file, &blocks).unwrap();
+    serde_json::to_writer(&mut file, &rtg_map).unwrap();
```
Comment (kostko) [inline genesis/bin/genesis_playback.rs]: Actually, are you sure this is correct? As far as I see the CLI tooling expects this to be wrapped in another dictionary under `runtime_states`.

Comment (kostko) [reply genesis/bin/genesis_playback.rs]: Oh nevermind.

File: genesis/bin/genesis_playback.rs (hunk 1 above)
Comment (matevz) [reply genesis/bin/genesis_playback.rs]: Yeah, this can safely be removed. Benchmarks are currently broken anyway, so we'll add this flag where necessary.

**Pull request merged**